src/style.css

```diff
@@ -0,0 +1,123 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
move it to the html selector

Reply:
you already moved font-family to html selector, delete here